5.5/examples/replica/README.md

```diff
@@ -26,8 +26,12 @@ This template will start with one MySQL master server and one slave server.
 ## Persistent storage
 
 In order to provide the persistent storage for the MySQL master server, the administrator
-of OpenShift needs to create a PersistentVolume that you can claim. This example requires a PersistentVolume of size 512m be available.
-To learn more about how to create PersistentVolume, refer to [OpenShift documentation](https://docs.openshift.org/latest/install_config/persistent_storage/persistent_storage_nfs.html)
+of OpenShift needs to create a PersistentVolume that you can claim. This example requires two persistent volumes of size 512m be available.
+To learn more about how to create PersistentVolume, refer to the [OpenShift documentation](https://docs.openshift.org/latest/install_config/persistent_storage/persistent_storage_nfs.html).
+
+The slave use persistent storage to serve as a hot backup and to optimize
+restart performance, such that only the data written while the slave was down
+needs to be replicated and not the entire database.
```

Comment:
"The slave template can be modified to use an EmptyDir volume to avoid the need for the additional persistent volume, at the cost of lengthier restart times due to the need to replicate the entire database to the slave when it restarts."